constraints.txt

```diff
@@ -19,6 +19,7 @@ funcsigs==1.0.2
 gitdb2==2.0.2
 github3.py==0.9.6
 GitPython==2.1.5
+humanize==0.5.1
```

Comment:
I should probably remove this, as I created separate constraints and requirements files for build-summary